lib/vrl/parser/src/lex.rs

```diff
@@ -1263,8 +1344,8 @@ mod test {
                                   bar""#);
 
         match lexer.next() {
-            Some(Ok((_, Token::StringLiteral(s), _))) => assert_eq!("foo bar", s.unescape()),
-            _ => panic!("Not a string literal"),
+            Some(Ok((_, StringLiteral(s), _))) => assert_eq!(TemplateString(vec![StringSegment::Literal("foo bar".to_string(), Span::new(1, 44))]), s.template(Span::new(0, 44))),
```

Comment:
nit: it'd be nice to adhere to 100 char limit (I guess rustfmt lets it slide because it's inside the `assert_eq!` macro).

Reply:
It's the `#[rustfmt::skip]` which is not needed here..

Reply:
Ah, yeah. I added that to a bunch of tests to make sure the `~` marks align with the code they are marking.